### ISSUE TITLE
Installation error using pip

### ISSUE THREAD

**Issue by arunppsg** [diffqc/dqc#15]
I am trying to reproduce the results of the paper Learning the exchange-correlation functional from nature with fully differentiable density functional theory and when I try to install `dqc` using pip in Ubuntu 20.04, I get an installation error.

Steps to reproduce:
1. Create a conda environment
2. Run `pip install dqc`

Error:
```
Defaulting to user installation because normal site-packages is not writeable
ERROR: Could not find a version that satisfies the requirement dqc (from versions: none)
ERROR: No matching distribution found for dqc
```

But installation via `git clone ..` works. Will it be possible to update the dqc in pypi registry?

**Comment by ferchault** [comment]
@arunppsg Which python version did you use for `git clone ..`? According to [PyPI](https://pypi.org/project/dqc/#files), precompiled versions of dqc are available until 3.9. A fresh install via `pip install git+https://github.com/diffqc/dqc` in a pristine python 3.11 environment failed due to a compilation error in numpy. 

For now we ended up having a separate conda environment with python 3.9 for dqc work.

**Comment by arunppsg** [comment]
I don't remember exactly but I remember using a separate conda env solved the issue for me.
